CITATION.cff

```diff
@@ -4,8 +4,8 @@ title: SageMath
 abstract: SageMath is a free open-source mathematics software system.
 authors:
 - name: "The SageMath Developers"
-version: 10.2.rc1
+version: 10.2.rc2
```

Comment:
Don't touch this file, it's updated automatically by the release manager, from CITATION.cff.in